src/data/int/basic.lean

```diff
@@ -376,10 +376,28 @@ match lt_trichotomy c 0 with
 | or.inr (or.inr hgt) := this hgt
 end
 
+protected theorem add_div_of_dvd_right (a b c : ℤ) (H : c ∣ b) :
```

Comment:
Also, this theorem should be moved after `add_mul_div_left`, to keep `right` and `left` variants together.
```suggestion
protected theorem add_div_of_dvd_right {a b c : ℤ} (H : c ∣ b) :
```

Reply:
Feel free to merge after moving this and accepting Johan's suggestions.

Thanks!
bors d+